api/v3/Order.php

```diff
@@ -73,61 +73,82 @@ function _civicrm_api3_order_get_spec(array &$params) {
  */
 function civicrm_api3_order_create(array $params): array {
   civicrm_api3_verify_one_mandatory($params, NULL, ['line_items', 'total_amount']);
-
+  if (empty($params['skipCleanMoney'])) {
```

Comment:
This is done when it gets to v3 contribution.create - but if we need to use this info we should do it here. We won't take cleaning money over to v4